### ISSUE TITLE
When using the msi internal UI with WixInternalUIBootstrapperApplication on high dpi monitor, the UI is too small

### ISSUE THREAD

**Issue by dcm99** [wixtoolset/issues#8538]
### WiX Version

5

### .NET or MSBuild or Visual Studio Version

.Net 8.0.300

### HeatWave Version

1.0.4

### Windows Version

Win10 22H2

### Repro Repo

https://github.com/dcm99/InternalUiBaDpiIssueExample

### Repro Steps

1. build project
2. start Bundle1.exe on an Monitor with 150% scaling or higher

### Actual Result

![grafik](https://github.com/wixtoolset/issues/assets/33775782/b430829e-b186-43f1-93a5-e817c2e1dfac)
the UI displays to small

### Expected Result

![grafik](https://github.com/wixtoolset/issues/assets/33775782/fcc3fc62-2e43-40ad-a0d4-e80268ba1922)
If you start the MSI File directly the scaling is correct

### Acknowledgements

- [X] I acknowledge that this is a fully completed bug report. It is not a question or attempt to get help debugging my issue (because those should be sent to Discussions).

**Comment by barnson** [comment]
Works as expected in Windows 11, fwiw.

**Comment by dcm99** [comment]
Thanks for taking a look at it.
After you couldn't recreate it on Windows 11, I wanted to recreate it today with my Windows 11 machine on the same monitor setup.
But before I copied the files from my Windows 10 machine, I tried it again on the same system and suddenly I can't recreate it.
The scaling of windows on Windows is weird, especially when using monitors with different DPI settings.
Sorry, next time I'll test on at least one other system before submitting an issue.
